sources/packages/api/src/database/entities/disbursement-feedback-errors.model.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from "typeorm";
+import { ColumnNames, TableNames } from "../constant";
+import { DisbursementSchedule } from "./disbursement-schedule.model";
+import { RecordDataModel } from "./record.model";
+
+/**
+ * Disbursement feedback errors codes for each schedule on a Student Application.
+ * coming from the E-Cert feedback file.
+ */
+@Entity({ name: TableNames.DisbursementFeedbackErrors })
+export class DisbursementFeedbackErrors extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+  /**
+   * Date received the error codes from the E-Cert feedback File.
+   */
+  @Column({
+    name: "date_received",
+    type: "timestamptz",
```

Comment:
Why do we specify here, the type of the column as timestamp with timezone? Is there a requirement that we need to capture it?

Reply:
date received is a time stamp,  `type: date ` will be like `2021-12-01 00:00:00.000+00`.  `type:timestamptz` will give 
![image](https://user-images.githubusercontent.com/77353155/144327671-25597b43-3b68-4818-9538-3fcdc5c5f565.png)

Reply:
Then I feel we need to create a story for changing it across the board.

Reply:
Type date means "date only" which would be completely wrong for this case.
I believe that what @guru-aot is saying is that, if you do not provide the type it is supposed to work in the same way.

Reply:
yup, without type will work, but for me, it made sense to add `type:timestamptz`, if @devs does not agree, then i can remove it

Reply:
I agree @ann-aot